cmd/wallet_import.go

```diff
@@ -19,6 +19,10 @@ import (
 var (
 	importWalletLong = cli.LongDesc(`
 		Import a wallet using the mnemonic.
+
+		The passphrase is used to protect the file in which the keys are stored. Hence, it
```

Comment:
Passphrase =/= mnemonic, or am I not interpreting this paragraph correctly

Reply:
The transition between the first and second sentence might be a bit too abrupt. I am just emphasizing the fact the passphrase is just used for file encryption so anything is fine, it's not tied to the mnemonic, nor the key generation.

Reply:
Ahh ok, how about something like "You will be asked to create a passphrase. The passphrase..."

Reply:
That's very good yes. Much clearer